internal/commands/app/create.go

```diff
@@ -183,14 +183,31 @@ func (cmd *CommandCreate) Handler(profile *user.Profile, ui terminal.UI, clients
 		return nil
 	}
 
+	// choose a data source to import template app schema data onto
```

Comment:
defaulting to the first cluster isn't ideal for anyone with more than one datasource. can we just add the prompt to let the user select?

Reply:
added a prompt if user is trying to link multiple data sources

Reply:
hmm now that im looking at it again, I think maybe it's better to move this to the _inputs.go file. I will do this tonight